Makefile

```diff
@@ -123,23 +123,23 @@ $(TARBALL):
 	$(GRAVITY) package export $(REPOSITORY)/$(NAME):$(VERSION) $(TARBALL) $(EXTRA_GRAVITY_OPTIONS)
 
 .PHONY: import
-import: images $(BUILD_DIR)/resources/app.yaml
+import: images
 	sed -i "s#gravitational.io/cluster-ssl-app:0.0.0+latest#gravitational.io/cluster-ssl-app:$(CLUSTER_SSL_APP_VERSION)#" resources/app.yaml
-	sed -i "s/tag: latest/tag: $(VERSION)/g" resources/charts/stolon/values.yaml
-	sed -i "s/0.1.0/$(VERSION)/g" resources/charts/stolon/Chart.yaml
+	sed -i "s/tag: latest/tag: $(VERSION)/g" resources/values.yaml
```

Comment:
`sed` will break if this is built from darwin. I'd suggest to use a macro to make it distro-independent:
```
define replace
	sed -i "$1" $2
endef

ifeq ($(OS),darwin)
define replace
	sed -i '' "$1" $2
endef
endif
```
and then use it:
```
$(call replace,"s/tag: $(VERSION)/tag: latest/g",resources/values.yaml)
```

Reply:
Is tele binary available for the Darwin architecture? I am not aware of it then. :(

Reply:
`tele` was (almost) always available for darwin. But even `gravity` works on darwin (with some minor changes)

Reply:
```
[ERROR]: tele build is not supported on darwin, only Linux is supported
```

Reply:
Yes, I know but it's trivial to remove an otherwise it should work. Sorry for confusion.